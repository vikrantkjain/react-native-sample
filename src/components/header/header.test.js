import React from 'react';
import renderer from 'react-test-renderer';
import Header from './index';

test('renders correctly', () => {
  const tree = renderer.create(<Header title="Test" type="home" />).toJSON();
  expect(tree).toMatchSnapshot();
});
