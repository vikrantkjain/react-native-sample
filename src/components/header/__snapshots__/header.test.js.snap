// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
[
  <RNCSafeAreaView
    edges={
      {
        "bottom": "off",
        "left": "off",
        "right": "off",
        "top": "additive",
      }
    }
    style={
      {
        "backgroundColor": "#fff",
        "flex": 0,
      }
    }
  />,
  <View
    style={
      {
        "backgroundColor": "transparent",
        "height": 64,
      }
    }
  >
    <View
      style={
        [
          {
            "flexDirection": "row",
            "paddingHorizontal": 20,
          },
          undefined,
        ]
      }
    >
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={false}
        hitSlop={
          {
            "bottom": 10,
            "left": 10,
            "right": 10,
            "top": 10,
          }
        }
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "justifyContent": "center",
            "opacity": 1,
          }
        }
      >
        <View
          style={
            [
              {
                "overflow": "hidden",
              },
              [
                {
                  "height": 17,
                  "padding": 10,
                  "tintColor": "blue",
                  "width": 10,
                },
                undefined,
              ],
            ]
          }
        >
          <FastImageView
            defaultSource={null}
            resizeMode="cover"
            source={
              {
                "testUri": "../../../src/assets/icons/ic_menu.png",
              }
            }
            style={
              {
                "bottom": 0,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          />
        </View>
      </View>
      <Text
        numberOfLines={1}
        style={
          [
            {
              "fontSize": 20,
              "marginLeft": 16,
            },
            undefined,
          ]
        }
      >
        Test
      </Text>
      <View
        style={
          {
            "height": 17,
            "padding": 10,
            "tintColor": "blue",
            "width": 10,
          }
        }
      />
    </View>
  </View>,
]
`;
